MovingBlocks/TerasologyLauncher
# Update Ukrainian locale with current strings

### Contains

As usual, updates the launcher's localization - this time around (since 2021) there's only one missing string to take care of :) This has been translated and converted to the fancy escape codes accordingly.

### How to test

Launch the launcher, switch the language to Ukrainian, then check out various UI screens to see if things look alright. Maybe look things over with Google Translate :)

### Outstanding before merging

None!
